### ISSUE TITLE
invisible main window

### ISSUE THREAD

**Issue by dingos84** [AhMyth/AhMyth-Android-RAT#209]
hello,
running the application on a virtual machine, the main window remains transparent.

**Comment by zlocate** [comment]
Yes. We need implementation standelone webserver without any Electron shit.
I think i do it in the near future at my fork (this repo like died)
